src/utility/util.py

```diff
@@ -0,0 +1,149 @@
+import os
+import textwrap
+from datetime import date, datetime, time
+from typing import Union
+
+from nextcord.ext import commands
+from nextcord.ext.commands import Group
+
+
+class NotImplementedYet(NotImplementedError):
+    def __init__(self):
+        super().__init__("This command has not been implemented yet.")
+
+
+def list_options(command: Union[Group, None]) -> str:
+    if command:
+        params = ", ".join(
+            f"`!!{cmd.qualified_name}`" for cmd in commands.Group.walk_commands(command)
+        )
+    else:
+        params = "No subcommands available."
+    return params
+
+
+def format_time(obj: time) -> str:
+    """
+    :return: string like "15:45:12"
+    """
```

Comment:
3 lines probably isnt necessary for 4 words
```suggestion
    """:return: string like `15:45:12`"""
```